schema/ridl/_example/example1-partials.ridl

```diff
@@ -68,13 +63,13 @@ service ExampleService
 
   - GetUser(req: GetUserRequest) => (user: User)
 
-  - Recv(req: stream string)
+  - stream Recv(req: string)
 
-  - Send() => (resp: stream string)
```

Comment:
Support for `stream` arguments is removed (we'll be able to stream responses or requests but it would be difficult to stream single arguments).